### ISSUE TITLE
rmob-export for Raspberry Pi4

### ISSUE THREAD

**Issue by meteorkees** [bolidozor/rmob-export#20]
Hi,
I am looking for a python module to upload my rmob.txt or rmob.dat file to the rmob.org website. I tried your script on a Pi4 but ran into a lot of errors, and I think the hole system has to work via a sort of Bolidozor server/network.

Is it possible to use your rmob-export script on a Pi4 with as a stand alone observer? (I use the Bolidozor V-dipool antenna to listen to GRAVES with a rtl-sdr v3 dongle)

Many thanks in advance,

Kees Habraken.
the Netherlands.

**Comment by kaklik** [comment]
Yes, rmob-export is not a primary stand-alone station-based application. Instead of this is a service for observers, to process the station's data uploaded by [data-uploader](https://github.com/MLAB-project/data-uploader) to the central Bolidozor network data server. 

Using rmob-export in a stand-alone station could be theoretically possible. But you should either use the same directory structure as-is on http://space.astro.cz/bolidozor/ or modify a fork of rmob-export to be better suited for your stand-alone use case.

**Comment by roman-dvorak** [comment]
Hi Kees, 

Thanks for the questions. The first thing is that the rmob generator has been programmed for use with python2. I'm not sure if it's currently compatible with python3 and the current version of the libraries.

With a few modifications, it would be possible to use our python colorgram generator. These modifications would be very welcome if you were able to help.

As kaklik wrote, the generator currently assumes a certain data structure. I dont know, how your data looks like. Coud you send a sample? 

The generator is based on one python class, which runs for each station (each station have own folder with measured data). Configuration for each station (it means each generated image) is possible to see here: http://space.astro.cz/bolidozor/svakov/SVAKOV-R12/rmob.cfg. It contains basic information. 

The main changes that you would need are:

* The first is to update it for the function with python3. 
* Second is to parametrize folder structure. Now it is hardcoded in python. Like [here](https://github.com/bolidozor/rmob-export/blob/c974dea0080b2e47da42855645a3ffa68acddae0/rmob.py#L231).
* Somehow to synchronize our data outputs form the detection software. 

If you are still interested in it, I can try it describe more specifically ang guide you.

**Comment by meteorkees** [comment]
Hello Roman and Jakub,
I will give it a go. The output from the MeteorRadio script is a .csv file.
This file is converted with another script to a observername_mmyyyyrmob.txt
file.
Only the .jpg file is still needed to be made, to get them uploaded to the
rmob.org server.
My son will take this job, and we think we have enough information to get
started.
The plan is to make a simple script that produces the rmob.jpg out of the
rmob.txt and get these 2 files uploaded to the rmob.org server.
Later this script can be implemented in whatever other python3 software.

We see how far we get for now, and if help is needed, we will ring the bell.

Best regards,

Kees.


Op za 4 jun. 2022 om 23:17 schreef Roman Dvořák ***@***.***>:
